lab7/src/main/java/agh/ics/oop/gui/App.java

```diff
@@ -0,0 +1,90 @@
+package agh.ics.oop.gui;
+
+import agh.ics.oop.*;
+import javafx.application.Application;
+import javafx.geometry.HPos;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.layout.ColumnConstraints;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.RowConstraints;
+import javafx.stage.Stage;
+
+public class App extends Application {
+    private AbstractWorldMap map;
+
+    @Override
+    public void init() throws Exception {
+
+        super.init();
+        String[] args = getParameters().getRaw().toArray(new String[0]);
+
+        try{
+            MoveDirection[] directions = new OptionsParser().parse(args);
+            //umieszczam zwierzątka i nimi nie poruszam
+            this.map = new GrassField(10);
+            this.map.place(new Animal(this.map, new Vector2d(2,2)));
+            this.map.place(new Animal(this.map, new Vector2d(0,0)));
```

Comment:
ok, ale fajnie by tu całą symulację odpalać, żeby móc to testować dla różnych danych łatwo